src/drivers/imu/mpu9250/mpu9250.cpp

```diff
@@ -404,7 +406,9 @@ MPU9250::read_reg16(unsigned reg)
 	uint8_t buf[2] {};
 
 	// general register transfer at low clock speed
-	_interface->read(MPU9250_LOW_SPEED_OP(reg), &buf, arraySize(buf));
+	if (_interface->read(MPU9250_LOW_SPEED_OP(reg), &buf, arraySize(buf)) != 2) {
+		PX4_WARN("MPU9250::read_reg16 failed");
```

Comment:
```suggestion
		PX4_DEBUG("MPU9250::read_reg16 failed");
```

Reply:
I think read_reg16 can be deleted entirely.

Reply:
Got.